lib/opentracing/reference.rb

```diff
@@ -76,13 +74,9 @@ def initialize(type, context)
     end
 
     # Return [String] reference type
-    def type
-      @type
-    end
+    attr_reader :type
```

Comment:
Can u please change the documentation line above, to be consistent with [YARD's documentation style for documenting attributes](https://github.com/lsegal/yard/blob/master/docs/GettingStarted.md#documenting-attributes).  Use `@return` instead of `Return`.